README.md

```diff
@@ -22,9 +22,54 @@ Metamask will generate a seed phrase for you. Write down this seed phrase and ke
 
-## 2. Clone this repo
+## 2. Get some test ETH
 
-## 3. Install Foundry
+It would be cool to eventually deploy some NFT's to the test network so you can see them for yourself!
+
+Ethereum has several testing networks that use fake ETH. We will be using the Goerli test network since that is being used by [Opensea](https://opensea.io/), an NFT marketplace.
+
+**Important: It is recommended you get at least 0.3 test ETH!** The network is congested and that means transaction fees are higher at this time.
+
+There are generally two ways to get test ETH:
+
+- Regular faucets
+- Mining yourself using POW (proof of work) faucets
+
+### Regular faucet
+
+1. Go to https://goerlifaucet.com/, and create an alchemy account. It doesn't matter what you put in there, you just need an alchemy account to claim the test ETH and it's free.
+
+2. Open up metamask, type your password and copy and paste your wallet address into the input field. This tells the faucet "Hey, send the funds over to my address!".
+
+3. Click "Send me ETH". In a short while, the funds should appear in your wallet. You will have to swap to the Goerli testnet in Metamask, though
+
+4. In Metamask, click on Settings > Advanced > Show test networks. Then, click on Goerli. Your funds should appear soon!
+
+### POW faucet
+
+1. Go to https://goerli-faucet.pk910.de/ (or search for Goerli POW faucet)
+2. Enter your wallet address, and start mining! **Note:** You need to mine for a certain amount of time because there is a minimum claim amount. Make sure you have the minimum claim amount before you claim your funds.
+3. When you've had enough, click "Stop mining and claim rewards"
+4. Verify that you are a human, and then your funds should appear shortly.
+
+**Tip**: If you have the resources, you can mine faster by increasing the amount of workers. You can increase this beyond what is initially shown as the max i.e. if it says 5/5 you can still increase the max workers if your machine allows it.
+
+Also, feel free to try out other faucets that might be faster.
+
+## 2. Install Brave Browser
+
+[Brave Browser](https://brave.com/) is a privacy-focused browser, but the main reason we are using this is because it is the easiest way to view files on [IPFS](https://docs.ipfs.tech/concepts/what-is-ipfs/).
+
+A small test:
+
+1. Open up Brave Browser
+2. Paste the following link => `ipfs://bafybeiapyyq2r2jb43qdyq3tzfyiqjxlamzlaa4xm7tfty4nzb6ol5e3sq`
+3. Choose gateway (not local node)
+4. If all went well, you should now be viewing the image of a [Bored Ape NFT](https://opensea.io/assets/ethereum/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d/2369)
+
+## 3. Clone this repo
+
+## 4. Install Foundry
```

Comment:
As we discussed i'd propose something like "Setting up your development environment"
Then start with both why NextJs and why foundry

Then proceed as you have done

Reply:
Also:
1.  i'd note that we use VScode for development. They are free to use other tools but we can only provide limited support there.  
2. If they open our repo from vsCode they should get a suggestion to install a set of extensions, we recommend them to do so